### PR TITLE
Implement Finnish i18n translation

### PR DESCRIPTION
**Pull request by tlindi**
Create i18n/fi.js initial version - FOSS ftw

### CODE REVIEW

**Comment by prusnak** [comment]
Please also run `make bundle` and add the result as the new commit with commit message: `chore: make bundle`

**Comment by tlindi** [comment]
> Please also run `make bundle` and add the result as the new commit with commit message: `chore: make bundle`

This was somewhat had for as I'm not experienced at all with poetry and such programming tools :-/ I tried to get "make" to complete. But unfor my skills lack. Best I got was

`"The currently activated Python version 3.8.9 is not supported by the project (^3.10 | ^3.9)." 
`
Unfortunately I don't have additional OS to create more recent compile environment and time is scarce too. Learnig curve is too steep for me. What to do?

**Comment by prusnak** [comment]
> What to do?

Okay, I did that for you and pushed to your PR branch.

**Comment by tlindi** [comment]
restored changes @prusnak had made and fixed one typo to make prittifier happy.

**Comment by prusnak** [comment]
> restored changes @prusnak had made and fixed one typo to make prettifier happy.

You did not. The file bundle.js is not part of your change. Why did you override my changes in the first place?

**Comment by tlindi** [comment]
> > restored changes @prusnak had made and fixed one typo to make prettifier happy.
> 
> You did not. The file bundle.js is not part of your change. Why did you override my changes in the first place?

Uuh.. by mistake. cause I'm so bad at this :'-( I'll stop now. Hopefully this translation some day finds it way to release.

**Review by prusnak — APPROVED** [review]

File: lnbits/templates/base.html
Comment (tlindi) [inline lnbits/templates/base.html:1]: is "," missing from KR line end
and extra one is at FI line end ?

Comment (prusnak) [reply lnbits/templates/base.html:1]: Good catch. Fixed

**Review by dni — APPROVED** [review]

**Comment by dni** [comment]
thanks :+1:

**Pull request merged**